src/main/java/net/sf/jabref/logic/formatter/bibtexfields/NormalizeNamesFormatter.java

```diff
@@ -43,20 +43,23 @@ public String getKey() {
     }
 
     @Override
-    public String format(String value) {
-        Objects.requireNonNull(value);
+    public String format(String nameList) {
+        Objects.requireNonNull(nameList);
         // Handle case names in order lastname, firstname and separated by ","
         // E.g., Ali Babar, M., Dingsøyr, T., Lago, P., van der Vliet, H.
-        if (!value.contains(" and ") && !value.contains("{") && !value.contains(";")) {
-            String[] valueParts = value.split(",");
+        if (!nameList.contains(" and ") && !nameList.contains("{") && !nameList.contains(";")) {
+            String[] arrayNameList = nameList.split(",");
             // Delete spaces for correct case identification
-            for (int i=0; i < valueParts.length; i++) {
-                valueParts[i] = valueParts[i].trim();
+            for (int i=0; i < arrayNameList.length; i++) {
```

Comment:
When you have list you can do a fancy one liner `originalStrings.replaceAll(String::trim);`
https://docs.oracle.com/javase/8/docs/api/java/util/List.html#replaceAll-java.util.function.UnaryOperator-